app/views/pages/code_of_conduct.html.erb

```diff
@@ -1 +1,75 @@
 <%= render PageTitleComponent.new(title: "Code of Conduct") %>
+
+<div class="flex flex-col text-justify max-w-3xl mx-auto gap-y-6 my-4">
+  <p>
+    <span class="text-wagon-red">Le Wagon</span> x <span class="text-other-green">Advent of Code</span> is about
+    learning, collaboration, sharing and having fun. It is a breeze of warmth of intellectual stimulation and kindness
+    as we approach Christmas.
+  </p>
+
+  <p>
+    During an event about fun, rules are boring, but boring people breaking rules is not fun.
+  </p>
+
+  <p class="strong">
+    All participants of the event are expected to abide by our Code of Conduct.
+  </p>
+
+  <ol class="list-decimal flex flex-col max-w-2xl mx-auto gap-y-4">
+    <li>
+      <h3 class="strong">Be respectful, show empathy towards community members</h3>
+      <p>
+        As obvious as it sounds, focus on good vibes and avoid any form of harrassment.
+      </p>
+    </li>
+    <li>
+      <h3 class="strong">Avoid spoilers</h3>
+      <p>
+        The community usually opens a thread in the
+        <%= link_to "#aoc Slack channel", Aoc.slack_channel, class: "link-explicit link-slack" %> every day to discuss
+        the puzzles. Try to avoid posting direct solutions in such threads.
```

Comment:
There were the "[SPOILERS] Solution thread for Day X" threads though, maybe worth mentioning them?

Reply:
Haha j'avais évoqué les 2 au début, puis j'ai changé d'avis (ça faisait un peu long). Mais si tu penses aussi que c'est worth je vais le remettre 👍